docs/src/00-doc/04_release_management.stories.mdx

```diff
@@ -26,11 +26,13 @@ It is also their responsibility to inform other relevant roles like UX and PM (i
 
 ## Creating a pre-release 
 
-If you want to be able to test components and features prior to their official release, you may create a pre-release. To make a pre-release you need to take the following steps:
- - To release a new version run the command `npm run bump-version -- [premajor | preminor | prepatch | prerelease]`. This will take care of updating the version number in all package.json files.
+If you want to be able to test components and features prior to their official release, you may create a pre-release. To make a pre-release you need to take the following steps: 
+ - Create a new branch from master and switch to it
+ - To release a new version, run the command `npm run bump-version -- [premajor | preminor | prepatch | prerelease]`. This will take care of updating the version number in all package.json files.
+ - Select `custom pre-release` and hit enter to use the default
```

Comment:
```suggestion
```

This doesn't fit to the previous step as it is written here. Probably, following the steps as outlined in this story and running `npm run bump-version -- prerelease` is the better way forward as it avoids typos and other mistakes if all that is supposed to happen is incrementing the number of the alpha release.

So, I would suggest deleting this line 32.